web-server/v0.4/src/pages/ComparisonSelect/index.js

```diff
@@ -266,7 +264,7 @@ class ComparisonSelect extends ReactJS.Component {
     }
 
     return (
-      <PageHeaderLayout title={selectedController}>
+      <PageHeaderLayout title={Array.isArray(selectedControllers) ? selectedControllers.join(', ') : selectedControllers}>
```

Comment:
When is it not an array?

Reply:
selectedControllers is not an array when the user follows the workflow of selecting a controller from the controllers view before navigating through the rest of the dashboard. We initialize selectedControllers as an array in the dashboard Redux model and overwrite it as a string when this case is met.

Reply:
Can we change that so that it is always an array, even if it is just one element?

Reply:
I've updated the data model to expect an array at every point we render the selected controllers.